addons/l10n_bf/data/account_tax_report_data.xml

```diff
@@ -315,7 +315,7 @@
                         <field name="code">BF_CREDIT_REPORTED</field>
                         <field name="expression_ids">
                             <record id="account_tax_report_line_bf_credit_reported_balance" model="account.report.expression">
-                                <field name="label">balance</field>
+                                <field name="label">_applied_carryover_balance</field>
```

Comment:
It's not enough :/ You need to keep another expression declared with label balance, else the column will never show this value in the report (because the column has expression_label = 'balance', so it only shows the values of expressions with name 'balance'). Basically, here, it'll just be an aggregation getting the value of _applied_carryover_balance

(multiple occurrences, in other files as well - replacing "balance" by "tax" when necessary)

Reply:
okey yeah make sense :)